src/fsi/src/monolithic/nonlinear_solver/4C_fsi_nox_linearsystem_gcr.cpp

```diff
@@ -136,7 +131,7 @@ bool NOX::FSI::LinearSystemGCR::applyJacobianInverse(
   }
 
   // Unscale the linear system
-  if (!!(scaling)) scaling->unscaleLinearSystem(Problem);
+  if (!!(scaling)) scaling->unscale_linear_system(Problem);
```

Comment:
```suggestion
  if (scaling) scaling->unscale_linear_system(Problem);
```

Reply:
Yep, there were a couple of other places like that, I fixed them too,